### PR TITLE
tests: add tests for `TextInput`

### PR DESCRIPTION
**Pull request by kamleshchandnani**
This PR adds tests for `TextInput`

- [x] Web tests
- [x] Native tests

### CODE REVIEW

File: packages/blade/src/components/Input/TextInput/__tests__/TextInput.web.test.tsx [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+        icon={InfoIcon}
+        prefix="https://"
+        suffix=".com"
+      />,
+    );
+    const prefix = getByText('https://');
+    expect(prefix).toBeVisible();
+    const suffix = getByText('.com');
+    expect(suffix).toBeVisible();
+    expect(container).toMatchSnapshot();
```
Comment (divyanshu013) [inline packages/blade/src/components/Input/TextInput/__tests__/TextInput.web.test.tsx:91]: Do we need snapshot for this?

**Review by divyanshu013 — APPROVED** [review]
Looks good to me.

File: packages/blade/src/components/Input/BaseInput/StyledBaseInput.web.tsx [hunk 2]
```diff
@@ -70,13 +70,15 @@ export const StyledBaseInput = React.forwardRef<HTMLInputElement, StyledBaseInpu
...
       setCurrentInteraction,
+      type,
       ...props
     },
     ref,
   ) => {
     return (
       <StyledBaseNativeInput
         ref={ref}
+        type={type === 'telephone' ? 'tel' : type}
```
Comment (divyanshu013) [inline packages/blade/src/components/Input/BaseInput/StyledBaseInput.web.tsx:81]: I think this would need a patch release?

Comment (kamleshchandnani) [reply packages/blade/src/components/Input/BaseInput/StyledBaseInput.web.tsx:81]: yes. will add changesets.

File: packages/blade/src/components/Input/TextInput/__tests__/TextInput.web.test.tsx (hunk 1 above)
Comment (kamleshchandnani) [reply packages/blade/src/components/Input/TextInput/__tests__/TextInput.web.test.tsx:91]: it has prefix, suffix, icon so if the icon size changes like we bump up the scale of icons internally then this snapshot will help us that the styling changed

**Review by divyanshu013 — APPROVED** [review]
Looks good to me.

**Pull request merged**